packages/gitgraph-react/src/Gitgraph.tsx

```diff
@@ -210,5 +211,32 @@ export class Gitgraph extends React.Component<GitgraphProps, GitgraphState> {
   }
 }
 
+// For now, this piece of logic is here.
+// But it might be relevant to move this back to gitgraph-core.
+// Ideally, it would be a method of Commit: `commit.message()`.
```

Comment:
👍 Totally agree with you about this comment, we should move this into gitgraph-core under `commit.message()`